public/button.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
```

Comment:
Great overall readability. This was well organized and formatting, and reads great compared to what you were doing a month ago!